### ISSUE TITLE
Clean up mythic score calculator

### ISSUE THREAD

**Issue by Thekinghim** [Thekinghim/keystone-companion#50]
I want it to have the same boarder size, same close button.

I want all the features to look identical so there is no difference in the UI style. 

I also want you to put more brain into the mythic score calculator.

**Comment by Larsj02** [comment]
we should probably make general frame designs to have everything identical like
General Frame (Main UI and Settings)
Popup Frame (Consumable Check, Talent check ...)
maybe more ?

**Comment by Thekinghim** [comment]
Yes that is what i mean but i also want the calculator to have more brain behind it.
